src/main/java/com/powsybl/openloadflow/ac/outerloop/AcLoadFlowParameters.java

```diff
@@ -54,6 +57,7 @@ public AcLoadFlowParameters(SlackBusSelector slackBusSelector, VoltageInitialize
         this.phaseControl = phaseControl;
         this.minImpedance = minImpedance;
         this.twtSplitShuntAdmittance = twtSplitShuntAdmittance;
+        this.breakers = breakers;
```

Comment:
If we don't keep the code to build LF network from IIDM bus/breaker view, we should not keep breakers option